### ISSUE TITLE
support scan coinbase transaction

### ISSUE THREAD

**Issue by mycute7** [turtlecoin/turtle-wallet-proton#66]
i have solo mined reward. but it seems that it does not support.

**Comment by ExtraHash** [comment]
no problem, I can hook that up easily

**Comment by Sajo8** [comment]
might be a good idea to have it disabled by default, tho

**Comment by ExtraHash** [comment]
yes, obviously

**Comment by ExtraHash** [comment]
Hi @mycute7 , thanks for making the issue. I've implemented a toggle button that's in the dev branch as of 458aef6 and will be on the next release.

However, I found out digging around in the code that @SoreGums actually hooked this up previously, there's just no toggle button in settings. So if you want to enable coinbase scanning in your current version, just open up the file in `~/.protonwallet/config.json` (or `C:/Users/username/.protonwallet/config.json in windows`) and change `scanCoinbaseTransactions: false,` to `scanCoinbaseTransactions: true,`
